<launch>
  <node pkg="jsk_2014_picking_challenge" type="fake_camera_info_publisher.py" name="fake_camera_info_publisher" output="screen"></node>
  <node pkg="jsk_2014_picking_challenge" type="image_publish_server.py" name="image_publish_server" output="screen"></node>

  <include file="$(find jsk_2014_picking_challenge)/launch/color_histogram_matcher.launch">
    <arg name="input_image" value="/image_publish_server/output" />
    <arg name="respawn" value="true" />
  </include>
</launch>

